### PR TITLE
ESSIFI-102: added dockerfile and docker-compose file

### PR DESCRIPTION
**Pull request by sksadjad**
added Dockerfile & docker-compose for generating image and publishing it

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+COPY sphereon-pe-models-*.tgz ./
+RUN npm install
+# If you are building your code for production
+# RUN npm ci --only=production
+
+# Bundle app source
+EXPOSE 3000
+COPY src ./src
+RUN cd src
+RUN npm start ./index.ts
```
Comment (maikel-maas) [inline Dockerfile:19]: The start script uses nodemon which is development mode. We need to transpile it first and then run it from the build folder.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+FROM node:14
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY package*.json ./
+COPY sphereon-pe-models-*.tgz ./
```
Comment (maikel-maas) [inline Dockerfile:10]: I think we can even remove this and pull it from npm remote repository

**Review by maikel-maas — CHANGES_REQUESTED** [review]
Almost there! :-)

**Comment by hrehman-sphereon** [comment]
We should be using PE-JS as dependency not PE-Models.
We should rename the project to pe-ms
The packages.json should have the name @sphereon/pe-ms
version 0.0.1
"license": "Apache-2.0",

Add unit tests
Add documentation to build , test , run seperately & run in docker.

remove hard coded token

**Review by hrehman-sphereon — CHANGES_REQUESTED** [review]
comments added already: Please have a look

File: README.md [hunk 3]
```diff
@@ -1 +1,52 @@
...
+The test command runs:
+* `eslint`
+* `prettier`
+* `unit`
+```shell
+nodemon src/index.ts
+```
+2. With docker
+
+You can simply start the service with running `docker-compose up` at the root of project
```
Comment (hrehman-sphereon) [inline README.md:52]: We should add more details about how to see it running. For example we need to add a link to swagger/postman workspace where we should have the ready examples to send request and see responses.

File: README.md [hunk 4]
```diff
@@ -1 +1,52 @@
...
+```shell
+yarn build
+```
+#### Test
+The test command runs:
+* `eslint`
+* `prettier`
+* `unit`
+```shell
+nodemon src/index.ts
```
Comment (hrehman-sphereon) [inline README.md:48]: I got this error on the first run: 'ts-node' is not recognized as an internal or external command, operable program or batch file.
We should add such dependencies command already in the README.

**Review by hrehman-sphereon — CHANGES_REQUESTED** [review]
There are some dependency related and README related comments

**Pull request merged**